src/static/css/2019.css

```
@@ -1348,6 +1361,45 @@ p.copyright a {
   min-width: 100%;
 }
 
+.nav-dropdown-list-part {
+  white-space: nowrap;
+  padding: 16px 8px 8px 8px;
+  font-weight: bold;
+  font-size: 1.25rem;
+}
+
+.nav-dropdown-list-part a {
+  padding: 0;
+}
+
+.nav-dropdown-list-part.foreword {
+  padding-top: 24px;
+}
+
+.nav-dropdown-list-chapter a,
+.nav-dropdown-list-chapter span {
+  padding: 8px 8px 8px 20px;
```

Extra padding as we've padded the part headers too:

```suggestion
  padding: 8px 8px 8px 24px;
```